src/lib/sql/sql_translator.cpp

```diff
@@ -640,6 +640,10 @@ SQLTranslator::TableSourceState SQLTranslator::_translate_table_origin(const hsq
                                         _with_descriptions, _meta_tables};
       lqp = subquery_translator._translate_select_statement(*hsql_table_ref.select);
 
+      // If a statement of the subquery is not cacheable (e.g., for meta tables),
+      // this statement should not be cacheable either.
+      cacheable = subquery_translator.cacheable;
```

Comment:
Shouldn't it be `|=`?

Reply:
If yes, there is probably a test missing.

Reply:
Agreed!

Reply:
Ah wait. It should be `&=`, right?

Reply:
Right, thank you! Only if outer and inner plan are cacheable, it should be set to `true`.